charts/redpanda/service_internal.go

```diff
@@ -0,0 +1,141 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +gotohelm:filename=service.internal.go.tpl
+package redpanda
+
+import (
+	"github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func MonitoringEnabledLabel(dot *helmette.Dot) map[string]string {
+	values := helmette.Unwrap[Values](dot.Values)
+	if values.Monitoring.Enabled {
+		return map[string]string{
+			// no gotohelm support for strconv.FormatBool
+			"monitoring.redpanda.com/enabled": "true",
```

Comment:
`fmt.Sprintf("%v", values.Monitoring.Enabled)` should work, right? There might be a more specific verb than `v` but I try to avoid reading the `fmt` docs because I just get angry at float formatting.